design-patterns/src/app/extensions/FormActionPatterns.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  Alert,
+  Box,
+  Button,
+  Flex,
+  Form,
+  Input,
+  Link,
+  Text,
+  TextArea,
+  hubspot,
+} from '@hubspot/ui-extensions';
+
+// Define the extension to be run within the Hubspot CRM
+hubspot.extend<'crm.record.tab'>(({ actions }) => (
+  <FormSimple actions={actions} />
+));
+
+// Define the FormSimple component.
+const FormSimple = ({ actions }) => {
+  const [formSubmitted, setFormSubmitted] = React.useState(false);
+
+  return (
+    <>
+      <Text>
+        This is some information about this form.{' '}
+        <Link href="https://www.hubspot.com">Learn more</Link>.
+      </Text>
+      {
+        // Display a success message when the form is submitted.
+        formSubmitted ? (
+          <Alert title="Success" variant="success">
+            Account created successfully.
+          </Alert>
+        ) : (
+          <Form
+            onSubmit={(e) => {
+              // Print out the form data when the form is submitted.
```

Comment:
I noticed the `actions` prop is also unused in this component. We can do 1 of 2 things:
1. Kill the `actions` prop in this example, or
2. Make use of `actions` in a way that's relevant to this example.

For # 2, one option could be to use [the `addAlert` action](https://developers.hubspot.com/docs/guides/crm/ui-extensions/sdk#display-alert-banners) to show a success message on submit, a la:
```js
const { first_name, last_name } = e.targetValue;
actions.addAlert({
  title: 'Success',
  variant: 'success',
  message: `Account created for ${first_name} ${last_name}`,
});
```